### PR TITLE
chore: Bump apptentive-kit-android from 6.0.3 to 6.1.0

### PR DESCRIPTION
**Pull request by ChaseApptentive**
## Summary
 - Update apptentive kit android version to 6.1.0

 ## Testing Plan
 - Smoke tested
 - Basic functionality test
 - Test app internal test track testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This repository contains the [Apptentive](https://www.apptentive.com/) integrati
 
     ```groovy
     dependencies {
-        implementation 'com.mparticle:android-apptentive-kit:5+'
+        implementation 'com.mparticle:android-apptentive-kit:6+'
```
Comment (markvdouw) [inline README.md:11]: Should remain at 5+

File: build.gradle [hunk 2]
```diff
@@ -49,6 +48,6 @@ allprojects {
 }
 
 dependencies {
-    api 'com.apptentive:apptentive-kit-android:6.0.3'
+    implementation 'com.apptentive:apptentive-kit-android:6.1.0'
```
Comment (markvdouw) [inline build.gradle:51]: Must remain at api instead of implementation

**Review by markvdouw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ChaseApptentive) [reply README.md]: Reverted

File: build.gradle (hunk 2 above)
Comment (ChaseApptentive) [reply build.gradle]: Changed back to api

**Comment by jshin-tse** [comment]
@markvdouw - Hey Mark, is there an ETA on this PR merge?

**Review by markvdouw — APPROVED** [review]

**Pull request merged**